.travis.yml

```diff
@@ -1,3 +1,16 @@
-script:
-- docker build --target all_tests -t test . && docker run --name todo-app-test --env TRELLO_URL --env TRELLO_KEY --env TRELLO_BOARD_ID --env TRELLO_TOKEN test
-- docker build --target prod -t prod .
+jobs:
+  include:
+    - stage: test
+      script:
+        - docker build --target all_tests -t test .
+        - docker run --name todo-app-test --env TRELLO_URL --env TRELLO_KEY --env TRELLO_BOARD_ID --env TRELLO_TOKEN test
+    - stage: deploy
+      script:
```

Comment:
This stage runs whenever any branch is updated.
That makes the `:latest` tag and the state of heroku unpredictable. It would be more useful and intuitive if you only deployed anything when the master branch is updated (and also only if the tests pass, but that's being handled already)